xeno/shield_removal.md

```diff
@@ -0,0 +1,94 @@
+## Abstract
+
+<!-- An abstract is a short blurb, about a paragraph or two, succinctly describing your feature. This should mostly be "why", but can include "what". -->
+
+Removing shields is something that's been requested for a long time. It will primarily involve removing shields from all castes (other than rav and crusher) and replacing them with other abilities or miscellaneous balance changes. There are a variety of detailed reasons explained in the reasons section below.
+
+Clarity stacks are limited effect stacks that let xenos resist minor stuns, can be seen on boxer warrior.
+
+Warden is a xeno that provides short-term buffs or heals. (not long term spam shields)
+
+Shaman is a xeno that provides short-term AOE buffs or AOE heals. (not long term shields for the entire hive such as in hijack)
+
+Warrior is a xeno that is hard to kill in a 1v1 and takes a frontliner / ambusher role with a preferably higher focus on frontlining.
+
+## Reasons
+
+Here I will go into more detail about the reasons to remove + overhaul shields. Copy pasted from my previous google doc.
```

Comment:
avoid referencing yourself in design docs

Reply:
done, expanded upon